kentikapi/internal/api_payloads/device.go

```diff
@@ -381,9 +403,19 @@ func payloadToDevicePlan(p devicePlanPayload) (models.DevicePlan, error) {
 		return models.DevicePlan{}, err
 	}
 
+	var id string
```

Comment:
This pattern with conditional conversion int to string repeats multiple times, maybe it's worth creating a helper function?